Automattic/jetpack
# Add Scheduled Updates /active endpoint

Context: p1714398995145929-slack-C01A60HCGUA

## Proposed changes:

* Removed the `active` field from the POST endpoints.
* Added new `/(?P<schedule_id>[\w]+)/active` endpoint.
* Adjusted unit tests according
* Fix: wrong check on `test_run_inactive_schedule`

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
* Please apply the PR using the `jetpack-downloader`
* Install D147343-code
* Follow instructions found here https://github.com/Automattic/jetpack/pull/37110
* In the https://developer.wordpress.com/docs/api/console/ make two calls to `POST /sites/__SITE__/update-schedules/__ID__/active?active=1` and `active=0`. Check that everything work as expected (the scheduled update should not run)
